src/linear_algebra/tensor_product.lean

```diff
@@ -237,7 +247,7 @@ instance : semimodule R (M ⊗ N) := semimodule.of_core
 
 variables (R M N)
 def mk : M →ₗ N →ₗ M ⊗ N :=
-linear_map.mk₂ R (⊗ₜ) add_tmul (λ c m n, by rw [smul_tmul, tmul_smul]) tmul_add tmul_smul
+linear_map.mk₂ R (⊗ₜ) (@add_tmul R _ _ _ _ _ _ _) (λ c m n, by rw [smul_tmul, tmul_smul]) tmul_add tmul_smul
```

Comment:
Any idea why this is necessary?

Reply:
Lean couldn't figure out `R`. I solved this by adding `[R]` in the tensor in the type.